instance/models/mixins/utilities.py

```diff
@@ -137,6 +137,46 @@ def _mail_admins_with_attachment(
 
 # Functions #####################################################################
 
+def send_urgent_deployment_failure_email(recipients: List[str], instance_name: str) -> None:
+    '''
```

Comment:
Can you use triple double quotes please (for the docstring)?

Reply:
Oh, sure. I overlooked this.